ha/core/system_health/const.py

```diff
@@ -71,6 +72,10 @@ class HEALTH_EVALUATOR_CLASSES:
 class CONFSTORE_KEY_ATTRIBUTES(Enum):
     STORAGE_SET_ID = "storage_set_id"
 
+class EVENT_ACTIONS(Enum, metaclass=EnumListMeta):
```

Comment:
This cannot be here, please check https://github.com/Seagate/cortx-ha/pull/522/

Reply:
Agree, rebased my code with fault_tolerance and used from there.